scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
I think it might make more sense that the min-width for this is the `$breakpoint-sm`, with the larger breakpoint for the font size 20px being the `$breakpoint-md`.  Sizing for this is a bit typography media query was done before we had formally decided on breakpoints, or having our media queries set up with "Mobile First" in mind.

As for the ???, I think the best way to maybe describe that might be "Default"  - it's at the $breakpoint-sm range where we we might want to start over-riding default styling with a media query.